File: 01. Life, The Universe And Everything/Solution.cpp
/**
  * STATS:
  *		Runtime - 0.0
  *		Memory - 4.3 MB
  *		Language - C++ 14 (GCC 8.3)
  */
  
#include <iostream>

int main() {
	// Disconnecting printf-scanf functions from cout-cin. This is a speed hack and will improve the performance of cin-cout drastically in many scenarios.
	std::ios_base::sync_with_stdio(false);
	
	// Running an infinite loop. Will break the loop from inside.
	while (true)	{
		// Creating a variable and taking input.
		static int input;
		std::cin >> input;
		
		// Checking for the break scenario. If the number is 42, breaking out of the loop.
		if (input == 42)
			break;
			
		// If the number isn't 42, printing the number. Using linefeed to end the line instead of `endl`. The latter will flush the buffer and waste time.
		std::cout << input << "\n";
	}
	
	return 0;
}

File: 01. Life, The Universe And Everything/Solution.java
/**
  * STATS:
  *		Runtime - 0.08
  *		Memory - 35 MB
  *		Language - Java (HotSpot 12)
  */

import java.util.*;
import java.lang.*;

class Solution {
	public static void main (String[] args) throws java.lang.Exception	{
		Scanner scanner = new Scanner(System.in);
 
		// Creating an infinite loop
		while (true)	{
			// Taking theinput using scanner.
			int input = scanner.nextInt();
 
			// Checking the break case, if the number is 42, breaking the infinite loop
			if (input == 42)
				break;
 
			// The flow-of-control will reach this point only if the input is not 42. So, simply printing the number.
			System.out.println(input);
		}
	}
}